content/pt/docs/what-is-opentelemetry.md

```diff
@@ -0,0 +1,133 @@
+---
+title: O que é OpenTelemetry?
+description:
+  Uma breve explicação sobre o que é o OpenTelemetry e o que ele não é.
+weight: 150
+default_lang_commit: f17277371622b97df4c15a6cfe3ce0b22e5538ef
+---
+
+O OpenTelemetry é uma estrutura de trabalho (_framework_) e um conjunto de
```

Comment:
I believe it was decided that we would not translate `framework` due to it being a well know/adopted term? 
```suggestion
O OpenTelemetry é um framework e um conjunto de
```

Reply:
I have to say I prefer to use `framework` instead of translating it. @joaopgrassi do we need to use it as italic then?

Reply:
I'd just follow the english one. If that uses it then we should use it here, otherwise not :)

Reply:
The `framework` is good for me, and I mentioned it on Slack about this. Framework brings more context than "work structure".